doc/README.open_ce_build.md

```diff
@@ -90,13 +90,15 @@ usage: open-ce build env [-h] [--conda_build_config CONDA_BUILD_CONFIG]
 positional arguments:
   env_config_file       Environment config file. This should be a YAML file
                         describing the package environment you wish to build.
-                        A collection of files exist under the envs directory.
+                        A collection of files exist at
+                        https://github.com/open-ce/open-ce-environments. This
```

Comment:
Can we also add an example of URL in the help?

Reply:
I agree - that's a good idea.

Reply:
Good idea. I improved the help for this argument.